frontend/packages/dev-console/integration-tests/support/pages/functions/checkDeveloperPerspective.ts

```diff
@@ -0,0 +1,15 @@
+export const checkDeveloperPerspective = (tries: number = 5) => {
+  if (tries < 1) {
+    return;
+  }
+  cy.get('body').then(($body) => {
+    cy.byLegacyTestID('perspective-switcher-toggle').click();
+    if ($body.find('[data-tour-id="perspective-switcher-menu-option"]').length !== 0) {
+      cy.byLegacyTestID('perspective-switcher-menu-option').contains('Developer');
+    } else {
+      cy.reload();
+      cy.document().its('readyState').should('eq', 'complete');
+      checkDeveloperPerspective(tries - 1);
```

Comment:
It's probably better just to wait for the rollout using CLI commands after the oc patch

Reply:
Sure, I will find the rollout command and update the code